src/test/java/org/broadinstitute/hellbender/tools/funcotator/FuncotatorIntegrationTest.java

```diff
@@ -597,5 +599,53 @@ public void testVcfDatasourceAccountsForAltAlleles() {
             Assert.assertEquals(funcotation.getField("dummy_ClinVar_VCF_CLNDISDB"), FuncotatorUtils.sanitizeFuncotationForVcf(gtString));
         }
     }
+
+    @Test
+    public void testCanAnnotateSpanningDeletions() {
+        final FuncotatorArgumentDefinitions.OutputFormatType outputFormatType = FuncotatorArgumentDefinitions.OutputFormatType.VCF;
+        final File outputFile = getOutputFile(outputFormatType);
+
+        final ArgumentsBuilder arguments = new ArgumentsBuilder();
+
+        arguments.addVCF(new File(SPANNING_DEL_VCF));
+        arguments.addOutput(outputFile);
+        arguments.addReference(new File(hg38Chr3Ref));
+        arguments.addArgument(FuncotatorArgumentDefinitions.DATA_SOURCES_PATH_LONG_NAME, DS_PIK3CA_DIR);
+        arguments.addArgument(FuncotatorArgumentDefinitions.REFERENCE_VERSION_LONG_NAME, FuncotatorTestConstants.REFERENCE_VERSION_HG38);
+        arguments.addArgument(FuncotatorArgumentDefinitions.OUTPUT_FORMAT_LONG_NAME, outputFormatType.toString());
+
+        runCommandLine(arguments);
+
+        final Pair<VCFHeader, List<VariantContext>> vcfInfo = VariantContextTestUtils.readEntireVCFIntoMemory(outputFile.getAbsolutePath());
+        final List<VariantContext> variantContexts = vcfInfo.getRight();
+        Assert.assertTrue(variantContexts.size() > 0);
+        final VCFHeader vcfHeader = vcfInfo.getLeft();
+        final VCFInfoHeaderLine funcotationHeaderLine = vcfHeader.getInfoHeaderLine(VcfOutputRenderer.FUNCOTATOR_VCF_FIELD_NAME);
+        final String[] funcotationKeys = extractFuncotatorKeysFromHeaderDescription(funcotationHeaderLine.getDescription());
+
+        final int NUM_VARIANTS = 10;
+        Assert.assertEquals(variantContexts.size(), NUM_VARIANTS);
+        Assert.assertEquals(variantContexts.stream().filter(v -> v.hasAllele(Allele.SPAN_DEL)).count(), NUM_VARIANTS);
+
+        for (int i = 0; i < variantContexts.size(); i++) {
+            final Map<Allele, FuncotationMap> alleleToFuncotationMap = FuncotatorUtils.createAlleleToFuncotationMapFromFuncotationVcfAttribute(funcotationKeys,
+                    variantContexts.get(i), "Gencode_28_annotationTranscript", "TEST");
+
+            // Make sure that all spanning deletions are funcotated with could not determine and that the others are something else.
+            for (final Allele allele : alleleToFuncotationMap.keySet()) {
+                if (allele.equals(Allele.SPAN_DEL)) {
+                    final List<String> txIds = alleleToFuncotationMap.get(allele).getTranscriptList();
+                    for (String txId : txIds) {
+                        if (allele.equals(Allele.SPAN_DEL)) {
+                            Assert.assertEquals(alleleToFuncotationMap.get(allele).get(txId).get(0).getField("Gencode_28_variantClassification"), GencodeFuncotation.VariantClassification.COULD_NOT_DETERMINE.toString());
```

Comment:
FYI, the `COULD_NOT_DETERMINE` classification was something I added for this kind of thing.  It's not an official value for `VariantClassification` according to the spec. 

I'm fine with using it, but it may warrant further discussion.

Reply:
I think it is fine to use. 

Done

Reply:
No action